LATEST

```diff
@@ -16,6 +16,7 @@ dbaded8b53da26fe183ea170c81bff2cac3deec8 2.5.2 SPLIT_RELEASE
 e0ad13da7621c94adbb16d4159d474dec16364c0 2.4.3 SPLIT_RELEASE
 cb6e0ca826880119e5d192ac0d9c0df9275c9084 2.4.2 SPLIT_RELEASE
 cf7c2b5cbf59dffcf61c93c502bf40e5a40bf6c0 2.4.0 SPLIT_RELEASE
+d0fa70c539ab8dda2fb70655a9b33ca4e3160029 2.3.17 SPLIT_RELEASE
```

Comment:
Because I forgot that commit existed and assumed there were no changes on the 2.3 branch without actually checking. Thanks for catching this!

In this specific case the missing commit is not critical as the GNU FTP was, in the end, only broken for a day or so, and is working again now. So for this one I'll just go ahead with d0fa70c539ab8.